### DISCUSSION TITLE
stoicism/william_rees/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/stoicism#16] [General]
# stoicism/william_rees/

https://shanenull.com/stoicism/william_rees/

**Comment by shane0** [comment]
# definitions

- problems have solutions
- predicaments have outcomes
- brain divided in 3 types under tension
- maladaptations

https://www.thegreatsimplification.com/episode/53-william-rees

https://mahb.stanford.edu/wp-content/uploads/2023/02/855-Article-Text-4479-1-10-20230131.pdf

**Reply by shane0** [reply]
# human exceptionalism

this is the phrase to describe people that live in denial that the they depend on an ecosystem

he was asked not to use the word animal when referring to humans

some people are so far in denial they reject the idea that they are an animal which is part of an ecosystem and evolved to live within the limits of the natural ecosystem

**Reply by shane0** [reply]
https://www.youtube.com/watch?v=T0PnsBfPn3o

**Reply by shane0** [reply]
https://www.youtube.com/watch?v=YBkl2kCdW-w

**Reply by shane0** [reply]
maladapted brains https://youtu.be/YBkl2kCdW-w

**Reply by shane0** [reply]
https://youtu.be/7h13Lm43sBM

**Comment by shane0** [comment]
# related

- https://www.fourthtransitionwealth.com/

**Reply by shane0** [reply]
## Louis Arnoux

- Our world is now passing through its thermodynamic peak and is on track to self-destruction. 
- Achieving the Fourth Transition, away from collapse, is the defining business opportunity of the 21st century. 
- It supersedes all others. It is also an existential necessity.
- https://www.fourthtransitionwealth.com/
- https://youtu.be/p9YCzrHugJI?si=yFF_4QYQJVM8LTZn
